README.md

```diff
@@ -34,6 +34,8 @@ Grow.
 
 ```
 extensions:
+  jinja2:
+  - extensions.kintaro.KintaroJinja2
```

Comment:
Name this `KintaroExtension` - we suffix `Extension` for the class name of all Jinja2 extensions.